src/overlays/actors/ovl_Bg_F40_Switch/z_bg_f40_switch.h

```diff
@@ -8,8 +8,14 @@ struct BgF40Switch;
 typedef void (*BgF40SwitchActionFunc)(struct BgF40Switch*, GlobalContext*);
 
 typedef struct BgF40Switch {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x20];
+    /* 0x0000 */ DynaPolyActor actor;
+    // TODO: signedness unknown, several unused
+    /* 0x015C */ s16 switchReleaseDelay; // frames until a pressed switch becomes released if nothing is still pressing it
+    /* 0x015E */ s8 isPressed; // Logical state of the switch (pressed or unpressed). Animation state may lag behind this slightly.
+    /* 0x015F */ s8 wasPressed; // used as temporary during update function
+    // true if this switch is the one that initiated a state change; false if this switch is changing as a result of another switch tied to the same flag.
+    // this is a temporary flag related to something currently happening, not a permanent property of the switch.
+    /* 0x0160 */ s8 isMaster;
```

Comment:
I don't like this name, but I haven't got a better suggestion for now.

Reply:
Maybe something like `isInitiator` or `initiated`?